### DISCUSSION TITLE
fatal: bad boolean config value 'null' for 'tag.forcesignannotated'

### DISCUSSION THREAD

**Discussion by Abd-Ur-Rehman** [Homebrew/discussions#1420] [Casks] [answered]
I'm wondering what could be wrong because when I install any app or run `brew update` then I'm getting the following error message.

```
fatal: bad boolean config value 'null' for 'tag.forcesignannotated'
fatal: bad boolean config value 'null' for 'tag.forcesignannotated'
```

What could be wrong or where can check I this bad boolean config to correct that? 

![image](https://user-images.githubusercontent.com/7280837/117449148-11c4ea80-af59-11eb-91bc-1906d71c8609.png)

**Comment by SMillerDev** [comment, marked as answer]
Your git config is broken, `tag.forcesignannotated` needs to be a boolean but it's `null`

**Reply by Abd-Ur-Rehman** [reply]
Sorry, I didn't get notice of that. I thought something wrong with it because recently had added the tp `teamookla/speedtest` I have added to `true` and it has fixed it. Thanks! :)
